src/common/ceph_timer.h

```diff
@@ -145,6 +146,7 @@ class timer {
 public:
   timer() : suspended(false) {
     thread = std::thread(&timer::timer_thread, this);
+    ceph_pthread_setname(thread.native_handle(), "ceph_timer");
```

Comment:
You might want to use `make_named_thread` from common/Thread.h (but you don't have to, either way is fine.)

Reply:
Thanks a lot for the review :-)
I saw that there are a few other wrappers to handle naming, but figured this was less intrusive. Happy to change it if somebody feels strongly about it though.